sdks/python/tox.ini

```diff
@@ -94,7 +94,7 @@ commands =
 # Checks for py2/3 compatibility issues
 deps =
   pycodestyle==2.3.1
-  pylint==1.7.2
+  pylint==1.7.6
```

Comment:
Should this match the pylint version above on line 83?

Reply:
There are 11 commits in this PR so you can look into the results of my efforts.